calico-cloud/image-assurance/scan-image-registries.mdx

```diff
@@ -153,6 +172,8 @@ Key names must match the full name of arguments passed to the CLI. The configura
 | --warn_threshold        | -w        | CC_WARN_THRESHOLD        | CVSS threshold for Warn scan results. Range from 0.0 - 10.0.                                                                |
 | --fail_threshold        | -f        | CC_FAIL_THRESHOLD        | CVSS threshold for Fail scan results. Range from 0.0 - 10.0.                                                                |
 | --vulnerability_db_path | -p        | CC_VULNERABILITY_DB_PATH | Path to a folder to store vulnerability data (defaults to `$XDG_CACHE_HOME`; if it is not set, defaults to `$HOME/.cache`). |
+| --input_file            | -i        | CC_INPUT_FILE            | Input file containing image URLs.                                                                                           |
```

Comment:
I think we should have some mention in the [export scan results section](https://github.com/tigera/docs/blob/main/calico-cloud/image-assurance/scan-image-registries.mdx#export-scan-results) that there is an option to export a batch of images and that file can then be fed into the CLI scanner using this input option.